Pseudocode/Graphs/Bipartite graph using DFS/SourceCode.tex

```diff
@@ -0,0 +1,68 @@
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Remove the 'end' at the end of the algorithm
+\usepackage[noend]{algpseudocode}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+\renewcommand{\algorithmicrequire}{\textbf{Input:}}
+\renewcommand{\algorithmicensure}{\textbf{Output:}}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+
+\begin{document}
+
+\begin{algorithm}
+
+  \caption{Check if a connected graph is bipartite using BFS}
+  
+  \algorithmicrequire \ A connected graph with V vertices\\
+  \algorithmicensure \ $true$ if the graph is bipartite, $false$ otherwise
+  
+  \begin{algorithmic}[1]
+    
+    \Statex
+    
+    \Function{$isBipartite$}{}
+    
+        \LeftComment{Initialization}
+        \State $Queue\ q$ \Comment{Used for the BFS}
+        \For {$i = 1 : V$} \Comment{Where each $i$ represents a vertex}
+            \State $color[i] \gets 0$
+            \State $visited[i] \gets false$
+        \EndFor
+        \State $q.push(0)$ \Comment{Index of the first node to explore}
+        \State $visited[0] \gets true$
+        
+        \Statex
+        \LeftComment{BFS}
+        \While{$q\ is\ \textbf{not\ empty}$}
+            \State $current \gets q.pop()$
```

Comment:
No need to use `()`. Moreover, it's better to be more explicit. `queue.front` and then `queue.pop` (as in some implementations, `q.pop` is void